packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 4.2.0 - Dec 20, 2022
+
+New features:
+
+- Use the new Dashboard tab template to create a personal and collaborative canvas in Teams. Learn more about this new template in the [documentation](https://aka.ms/teamsfx-dashboard-app).
+
+Enhancements:
+- Step-by-step guidance in Visual Studio Code to assisit you while creating a free Microsoft 365 testing tenant.
```

Comment:
Looks like there is a spelling error of `assist`.

Reply:
Would it be possible to link to a screenshot of the dashboard template under our description?

Reply:
We had also talked about Dashboard being a sample and not a template, but it seems like perhaps the ship has sailed on that?

Reply:
Updated screenshot and fixed typo.